flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/stream/sql/WindowRankTest.xml

```diff
@@ -212,8 +212,8 @@ LogicalProject(a=[$0], b=[$1], c=[$2], d=[$3], e=[$4], rowtime=[$5], proctime=[$
     </Resource>
     <Resource name="optimized rel plan">
       <![CDATA[
-Calc(select=[a, b, c, d, e, rowtime, PROCTIME_MATERIALIZE(proctime) AS proctime, window_start, window_end, window_time, rownum])
-+- WindowRank(window=[CUMULATE(win_start=[window_start], win_end=[window_end], max_size=[1 h], step=[10 min])], rankType=[ROW_NUMBER], rankRange=[rankStart=1, rankEnd=3], partitionBy=[a], orderBy=[b DESC], select=[a, b, c, d, e, rowtime, proctime, window_start, window_end, window_time, rownum])
+Calc(select=[a, b, c, d, e, rowtime, PROCTIME_MATERIALIZE(proctime) AS proctime, window_start, window_end, window_time, w0$o0])
```

Comment:
In the previous materialize version, the last field of the new added project is rowNum
![image](https://user-images.githubusercontent.com/1525333/128148720-30eec7cb-3080-45bf-803c-a9d24dcb4505.png)

In the new version, materialize move to the later phase. after `project_rewrite` phase, the last field of LogicalProject rowtype is $5 instead of rowNum.
So  the last field of the new added calc is $5
![image](https://user-images.githubusercontent.com/1525333/128148392-59d0f520-db2e-40d7-8ee7-e8b6038eacb9.png)